components/Account/UpdateMe.js

```diff
@@ -150,10 +151,10 @@ class UpdateMe extends Component {
     const errorMessages = Object.keys(errors)
       .map(key => errors[key])
       .filter(Boolean)
-
     return (
       <Loader loading={loading || !me} error={error} render={() => (
         <div style={style}>
+          <UpdateEmail t={t} me={me} />
```

Comment:
nit: you already have `t` from `withT`, use `withMe` in `UpdateEmail` and make it completely prop less.

Reply:
Will that not trigger an unnecessary query?

Reply:
apollo does query de-deup—will just load it from the in-memory cache

Reply:
And I'm not sure whether "sign out via refetch" still would work.